hack/update_fbc.sh

```diff
@@ -0,0 +1,75 @@
+# Helper tool to update the catalog artifacts and bundle artifacts
+# These genereated artifacts are used to build the catalog image
+# Pre-requisites: opm, make
+# Usage: ./hack/update_bundle_catalog.sh
+
+#!/bin/bash
+
+set -euo pipefail
+
+cleanup() {
+  # remove temporary bundle file
+  if [ -n "${TEMP_BUNDLE_FILE}" ]; then
+    rm -f "${TEMP_BUNDLE_FILE}"
+  fi
+
+}
+
+trap cleanup EXIT
+
+: ${OPM:=$(command -v opm)}
+echo "using opm from ${OPM}"
+# check if opm version is v1.39.0 or exit
+if [ -z "${OPM}" ]; then
+  echo "opm is required"
+  exit 1
+fi
+
+: ${YQ:=$(command -v yq)}
+echo "using yq from ${YQ}"
+# check if yq exists
+if [ -z "${YQ}" ]; then
+  echo "yq is required"
+  exit 1
+fi
+
+: "${CATALOG_FILE:=catalog/index.yaml}"
+
+cat <<EOF >"${CATALOG_FILE}"
+---
+defaultChannel: latest
+icon:
```

Comment:
What icon are you using?  Does it need to be embedded in the script like this, or could you upload and point to the svg file?

Reply:
this bpfman log and it can be verified using https://base64.guru/converter/decode/image
and other operators doing the same the also embedded that way so we are following alone

Reply:
I used the one from the bundle:
https://github.com/bpfman/bpfman-operator/blob/c596fe23739ce9d7950fe6cd9f48543b9ad640b5/bundle/manifests/bpfman-operator.clusterserviceversion.yaml#L407

I can include it in a separate file and make the script use it if you prefer.

Reply:
no that is good enough Thanks

Reply:
This is good.  Thanks.